### PR TITLE
chore: Set proper limits for the redesigned HBAR limiter 

### LINKED ISSUE #2895: [HBAR Rate Limit Redesign] Set proper daily limits for subscription tiers

### PR DESCRIPTION
**Pull request by victor-yanev**
**Description**:

Calculated using the formula:
```math
(Total \space budget * percentage \space of \space budget \space allocated \space to \space tier) / amount \space of \space plans \space expected \space for \space tier
```
- TOTAL: 8000 HBARs
- BASIC: 11.2 HBARs = (8000 * 0.7) / 500
- EXTENDED: 32 HBARs = (8000 * 0.2) / 50
- PRIVILEGED: 80 HBARs = (8000 * 0.1) / 10

**Related issue(s)**:

Fixes #2895 

**Notes for reviewer**:

**Checklist**

- [x] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

File: packages/relay/src/lib/constants.ts [hunk 1]
```diff
@@ -140,11 +140,11 @@ export default {
   },
 
   // TODO: Replace with actual values - https://github.com/hashgraph/hedera-json-rpc-relay/issues/2895
```
Comment (quiet-node) [inline packages/relay/src/lib/constants.ts:142]: remove todo

File: packages/relay/src/lib/constants.ts [hunk 2]
```diff
@@ -140,11 +140,11 @@ export default {
...
-  HBAR_RATE_LIMIT_DURATION: parseInt(process.env.HBAR_RATE_LIMIT_DURATION || '80000'), // 80 seconds
-  HBAR_RATE_LIMIT_TOTAL: BigNumber(process.env.HBAR_RATE_LIMIT_TINYBAR || '11000000000'), // 110 HBARs per 80 seconds
-  HBAR_RATE_LIMIT_BASIC: BigNumber(process.env.HBAR_RATE_LIMIT_BASIC || '92592592'), // Equivalent of 1000 HBARs per day
-  HBAR_RATE_LIMIT_EXTENDED: BigNumber(process.env.HBAR_RATE_LIMIT_EXTENDED || '925925925'), // Equivalent of 10000 HBARs per day
-  HBAR_RATE_LIMIT_PRIVILEGED: BigNumber(process.env.HBAR_RATE_LIMIT_PRIVILEGED || '1851851850'), // Equivalent of 20000 HBARs per day
+  HBAR_RATE_LIMIT_DURATION: parseInt(process.env.HBAR_RATE_LIMIT_DURATION || '86400000'), // 1 day
+  HBAR_RATE_LIMIT_TOTAL: BigNumber(process.env.HBAR_RATE_LIMIT_TINYBAR || '800000000000'), // 8000 HBARs
```
Comment (quiet-node) [inline packages/relay/src/lib/constants.ts:144]: Here we use `HBAR_RATE_LIMIT_TOTAL` but in the doc it's `HBAR_RATE_LIMIT_TINYBAR`. I like `HBAR_RATE_LIMIT_TOTAL` better.

File: docs/configuration.md [hunk 3]
```diff
@@ -67,11 +67,11 @@ Unless you need to set a non-default value, it is recommended to only populate o
...
-| `HBAR_RATE_LIMIT_PRIVILEGED`                | "1_851_851_850"            | Individual limit in tinybars for spending plans with a PRIVILEGED tier. (equivalent of 20_000 HBARs per day for a duration of 80 seconds)                                                                                                                                                              |
+| `HBAR_RATE_LIMIT_DURATION`                  | "86400000"                 | HBAR budget limit duration. This creates a timestamp, which resets all limits, when it's reached. Defaults to 1 day.                                                                                                                                                                                   |
+| `HBAR_RATE_LIMIT_TINYBAR`                   | "800000000000"             | Total HBAR budget (in tinybars). Defaults to 8000 HBARs.                                                                                                                                                                                                                                               |
```
Comment (quiet-node) [inline docs/configuration.md:71]: Rename HBAR_RATE_LIMIT_TINYBAR to be HBAR_RATE_LIMIT_TOTAL

**Comment by quiet-node** [comment]
@victor-yanev  
Q: Did I miss the conversation somewhere on how we came up with that formula and those numbers? Or were they just numbers laid out for discussion?

Also, if those numbers have been discussed and agreed upon, let’s not forget to include them in the design docs as well.

File: packages/relay/src/lib/constants.ts (hunk 2 above)
Comment (Nana-EC) [reply packages/relay/src/lib/constants.ts:143]: Unfortunately we can't really rename or replace configs easily due to backwards compatibility.
We should actually change this back `HBAR_RATE_LIMIT_TINYBAR` if `HBAR_RATE_LIMIT_TOTAL` was never used before.
The issue is if an operator has configured it today and upgrades we don't wan to break them by no longer respecting the previous config.

**Review by Nana-EC — APPROVED** [review]

File: packages/relay/src/lib/constants.ts (hunk 2 above)
Comment (quiet-node) [reply packages/relay/src/lib/constants.ts:143]: Right yeah that makes sense!

**Review by ebadiere — APPROVED** [review]

File: packages/server/tests/acceptance/htsPrecompile/tokenCreate.spec.ts [hunk 4]
```diff
@@ -243,9 +243,6 @@ describe('@tokencreate HTS Precompile Token Create Acceptance Tests', async func
   });
 
   it('should associate to a token with custom fees', async function () {
-    //delay for hbar rate limiter to reset
```
Comment (quiet-node) [inline packages/server/tests/acceptance/htsPrecompile/tokenCreate.spec.ts:246]: With the new limiter this is no longer relavant.

File: packages/server/tests/acceptance/htsPrecompile/tokenManagement.spec.ts [hunk 5]
```diff
@@ -181,9 +181,6 @@ describe('@tokenmanagement HTS Precompile Token Management Acceptance Tests', as
     let tokenAddress, tokenContract, nftAddress;
 
     before(async function () {
-      //delay for hbar rate limiter to reset
```
Comment (quiet-node) [inline packages/server/tests/acceptance/htsPrecompile/tokenManagement.spec.ts:184]: With the new limiter this is no longer relavant.

**Review by quiet-node — APPROVED** [review]

**Review by ebadiere — APPROVED** [review]

**Review by Nana-EC — APPROVED** [review]

**Pull request merged**